modules/setup/library/Setup/WebWizard.php

```diff
@@ -711,7 +709,23 @@ public function getRequirements($skipModules = false)
                 'setup.requirement.class'
             )
         )));
-        $set->merge($pgsqlSet);
+        $dbSet->merge($pgsqlSet);
+        $set->merge($dbSet);
+
+        $dbRequire = (new SetRequirement(array(
+            'optional'      => false,
+            'condition'     => $dbSet,
+            'title'         =>'Database',
+            'alias'         => 'PDO-MySQL OR PDO-PostgreSQL',
+            'description'   => mt(
+                'setup',
+                'To store users or preferences in a database at least one module PDO-MySQL OR PDO-PostgreSQL for PHP 
```

Comment:
Please don't mention the reason why a database module is required. On one hand it's not only the user preferences but also users themselves, on the other hand both reasons are mentioned directly above. So I think we can reduce this to mentioning that a database is mandatory and hence these modules are required.